users/openapi.yml

```diff
@@ -66,6 +66,25 @@ paths:
           description: Missing or invalid access token provided.
         '500':
          $ref: "#/components/responses/ServiceError"
+  /groups/{groupID}:
+    get:
+      summary: Get users that belongs to a group
```

Comment:
Do we need this summary?

Reply:
there is always this summary, I guess yes